tests/suites/test_suite_sm3.data

```diff
@@ -0,0 +1,18 @@
+SM3 Valid parameters
+sm3_valid_param:
+
+SM3 Test Vector A.1.  GB/T 32918.2-2016 Example 1
+depends_on:MBEDTLS_SM3_C
+mbedtls_sm3:"616263":"66c7f0f462eeedd9d1f2d46bdc10e4e24167c4875cf2f7a2297da02b8f4ba8e0"
+
+SM3 Test Vector B.1.  GB/T 32918.2-2016 A.2 Example 1
```

Comment:
Add more example vectors

Reply:
I will update mbedtls SM3 test suite with the same test vectors used for PSA, thanks.